### ISSUE TITLE
Add a default Bazel BUILD file

### ISSUE THREAD

**Issue by boutblock** [onqtam/doctest#433]
Please include a Bazel BUILD file (https://bazel.build/) at the root of the generated archive containing:
```
load("@rules_cc//cc:defs.bzl", "cc_library")

cc_library(
    name = "doctest",
    hdrs = glob(["doctest/**/*.h"]),
    visibility = ["//visibility:public"],
    includes = ["doctest/"],
)
```
So that it can be easily downloaded from github and build lcoally with a simple 'http_archive' command.

**Comment by onqtam** [comment]
I'd be happy to accept a pull request :)

**Comment by onqtam** [comment]
fixed in #497 !
